projects/lib/src/dynamic-views/zone/zone.component.html

```diff
@@ -1,43 +1,35 @@
 <ng-container
-*ngTemplateOutlet="isArray ? arrayTpl : zoneTpl; context: { $implicit: data, first: 'true' }"
+*ngTemplateOutlet="isArray ? array : zone; context: { $implicit: zoneContext !== null ? zoneContext.data : undefined, first: 'true' }"
 ></ng-container>
 
 <!-- data is an array -->
-<ng-template #arrayTpl let-data>
+<ng-template #array let-data>
   <ng-container *ngFor="let d of data; let index = index; let first = first"
-    [ngTemplateOutlet]="zoneTpl"
+    [ngTemplateOutlet]="zone"
     [ngTemplateOutletContext]="{ $implicit: data, index, first }">
```

Comment:
I think it's not necessary to inject `data` into the child template, because it's not used anymore, except in `onItemClicked()`, but you could just access the data object from zoneContext.